components/reference-data/src/main/java/org/datacleaner/beans/transform/SynonymLookupTransformer.java

```diff
@@ -156,7 +156,11 @@ public void close() {
         final String originalValue = inputRow.getValue(column);
 
         if (originalValue == null) {
-            return new String[1];
+            if (lookUpEveryToken) {
+                return new String[3];
```

Comment:
Not really related, but this reminds me that during a demo, it was a bit of a pain that we don't have the synonyms found and masterterm found column when not doing every "token" (I guess that word is actually wrong in some cases now).

Would there be any drawbacks to always returning them?